integration/spark-common/src/main/scala/org/apache/carbondata/spark/rdd/CarbonScanRDD.scala

```diff
@@ -116,8 +116,8 @@ class CarbonScanRDD(
           i += 1
           result.add(partition)
         }
-      } else if (sparkContext.getConf.contains("spark.carbon.custom.distribution") &&
-                 sparkContext.getConf.getBoolean("spark.carbon.custom.distribution", false)) {
+      } else if (java.lang.Boolean
+        .parseBoolean(CarbonProperties.getInstance().getProperty("carbon.custom.distribution"))) {
```

Comment:
The PR's title mentions that the property is "spark.carbon.custom.distribution", but here you change the property name to "carbon.custom.distribution", why ?